src/workloads/networking/CommitLatency.yml

```diff
@@ -63,214 +63,98 @@ Actors:
 
 
 
-# All of the below is really just the same actor, but as Genny reports statistics using the actor
-# name, we need a different actor instance for each configuration. All of the below run in their
-# own slot, nothing is concurrent.
-# Since Drop+Reload is in phases 0,1,-,3,4,-... then the test itself must always be in a slot at
-# 2,5,8... = 3n-1 = 3n+2
-- Name: CommitLatency_w0_local
+- Name: CommitLatency
   Type: CommitLatency
   Threads: 1
   Repeat: 500
   Database: test
   Phases:
-  - Phase: 0..1
-    Operation:
-      OperationName: Nop
+  - *Nop # Load phase is different actor
+  - *Nop
   - WriteConcern:
       Level: 0
     ReadConcern:
       Level: local
     ReadPreference:
       ReadMode: primary
-  - Phase: 3..32
-    Operation:
-      OperationName: Nop
-
-- Name: CommitLatency_w1_local
```

Comment:
I think we should allow phases to have names instead of appending just the phase number. In this case, the phase number can be `w1_local`. The metric name would then be: `[actor-name] << "." << [phase-name]`. Phases with the same name can still have their metrics be grouped together.

Reply:
> I think we should allow phases to have names instead of appending just the phase number. 

Feel free to add to [Genny Ideas](https://docs.google.com/document/d/1K0SlHoKkIACIXhkNP_KPHvb3IV6r53ReqsHqEYTnXDA/edit#) but each Actor may think of each Phase differently. 

I think the recommendation is for an Actor to be configurable with the metrics name for the Phase:

```yaml
Actors:
- Type: MyActor
  Phases:
  - Phase: 0
    MetricsName: Insert
```